helm/adelphi/templates/gemini-job.yaml

```diff
@@ -42,12 +42,14 @@ spec:
       - name: results-pv
         mountPath: /results
     initContainers:
-    - name: download-gemini
-      image: alpine
-      command: ["sh", "-c"]
-      workingDir: /workspace
+      # remove this container when Adelphi is available in PyPI
+    - name: clone-adelphi-cli
+      image: alpine/git
+      command:
+        - sh
+        - -c
       args:
-        - wget -O - {{ .Values.gemini_download_url }} | tar -zxvf -
+        - git clone https://github.com/datastax/adelphi.git /workspace/adelphi
```

Comment:
temporary solution while Adelphi is not in PyPI

Reply:
Now that we're there presumably all of this can come out... ?

Reply:
Will address this in small follow-up PR